piwik-sdk/src/test/java/org/piwik/sdk/TrackerTest.java

```diff
@@ -248,7 +227,7 @@ public void testGetResolution() throws Exception {
         Tracker tracker = createTracker();
         TrackMe trackMe = new TrackMe();
         tracker.track(trackMe);
-        assertTrue(trackMe.build().contains("res=480x800"));
+        assertTrue(trackMe.build(), trackMe.build().contains("res=480x800"));
```

Comment:
The values returned are somehow 0x0, not sure why that suddenly is. I saw no changes regarding that.